testplan/common/entity/base.py

```diff
@@ -188,6 +188,14 @@ def all_status(self, target):
             for resource in self._resources
         )
 
+    def _fetch_log_messages(self, resource, message):
```

Comment:
I do not think this functions should have a role to take a prefix message and append to it, this just should format the part that is the log message and format it, do the concatenation where you prepre the message.

let this has a single role